e2e/portalicious/tests/SendMessage/SendCustomMessage.spec.ts

```diff
@@ -0,0 +1,55 @@
+import { test } from '@playwright/test';
+
+import { SeedScript } from '@121-service/src/scripts/seed-script.enum';
+import { seedIncludedRegistrations } from '@121-service/test/helpers/registration.helper';
+import { resetDB } from '@121-service/test/helpers/utility.helper';
+import { getAccessToken } from '@121-service/test/helpers/utility.helper';
+import { registrationsPV } from '@121-service/test/registrations/pagination/pagination-data';
+
+import BasePage from '@121-e2e/portalicious/pages/BasePage';
+import LoginPage from '@121-e2e/portalicious/pages/LoginPage';
+import RegistrationsPage from '@121-e2e/portalicious/pages/RegistrationsPage';
+
+const customMessageText = 'This is a custom message from the Red Cross.';
+const sendingMessageToast =
+  'Closing this notification will not cancel message sending.';
+
+test.beforeEach(async ({ page }) => {
+  await resetDB(SeedScript.nlrcMultiple);
+  const programIdPV = 2;
+  const pvProgramId = programIdPV;
+
+  const accessToken = await getAccessToken();
+  await seedIncludedRegistrations(registrationsPV, pvProgramId, accessToken);
+
+  // Login
+  const loginPage = new LoginPage(page);
+  await page.goto('/');
+  await loginPage.login(
+    process.env.USERCONFIG_121_SERVICE_EMAIL_ADMIN,
+    process.env.USERCONFIG_121_SERVICE_PASSWORD_ADMIN,
+  );
+});
+
+test('[31077] Send custom message', async ({ page }) => {
+  const basePage = new BasePage(page);
+  const registrations = new RegistrationsPage(page);
+
+  const projectTitle = 'NLRC Direct Digital Aid Program (PV)';
+
+  await test.step('Select program', async () => {
+    await basePage.selectProgram(projectTitle);
+  });
+
+  await test.step('Send custom message', async () => {
+    await registrations.selectAllPAs();
+    await registrations.selectBulkAction('Message');
+    await registrations.selectCustomMessage();
+    await registrations.typeMessage(customMessageText);
+    await registrations.clickContinueToPreview();
+    await registrations.validateMessagePresent(customMessageText);
+    await registrations.sendMessage();
+
+    await registrations.validateToastMessage(sendingMessageToast);
```

Comment:
I think we should also validate that the message has been sent. ie. navigate to the activity log and check that the message is in the activity log. Do you think we can add that?

Reply:
Added